PCbuild/rt.bat

```diff
@@ -34,7 +34,7 @@ set pyname=python
 set suffix=
 set qmode=
 set dashO=
-set regrtestargs=--fast-ci
+set regrtestargs=--fast-ci -u-gui
```

Comment:
Buildbots run `Tools/buildbot/test.bat` which runs `PCbuild/rt.bat`. I'm not sure that this change is correct. Maybe it will disable GUI tests on buildbots.

Reply:
Yes, thanks for this question. I was not also sure about that. Since I don't ever use windows and not quite sure about how it should work. I will revert this.